litellm/llms/prompt_templates/factory.py

```diff
@@ -1346,6 +1346,16 @@ def prompt_factory(
                 return anthropic_pt(messages=messages)
         elif "mistral." in model:
             return mistral_instruct_pt(messages=messages)
+        elif "llama2" in model:
+            return llama_2_chat_pt(messages=messages)
+        elif "llama3" in model:
```

Comment:
can you check if it's an 'instruct' model before applying the chat template (same for llama2).

Reply:
Bedrock only supports Llama-2-Chat and Llama-3-Instruct models [1].

Nonetheless, I think it still makes sense to add a check just to be future proof. Will update.

 [1] https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html#model-ids-arns

Reply:
i see non-chat llama2 models on bedrock

<img width="1134" alt="Screenshot 2024-04-25 at 11 20 10 AM" src="https://github.com/BerriAI/litellm/assets/17561003/1a138a2e-97ff-4bac-84bc-855b527fe437">